<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSA24Class</class>
 <widget class="QMainWindow" name="DSA24Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSA24</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>1</y>
      <width>611</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTableWidget" name="playListTable">
       <column>
        <property name="text">
         <string>Title</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Artist</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Duration</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Link</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>START</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string>NEXT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="prevButton">
           <property name="text">
            <string>PREVIOUS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="playButton">
           <property name="text">
            <string>PLAY</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="songList"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>370</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="titleLabel">
             <property name="text">
              <string>TITLE</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="artistLabel">
             <property name="text">
              <string>ARTIST</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="durationLabel">
             <property name="text">
              <string>DURATION</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="linkLabel">
             <property name="text">
              <string>LINK</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPlainTextEdit" name="titleBox"/>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="artistBox"/>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="durationBox"/>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="linkBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>REMOVE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DSA24.qrc"/>
 </resources>
 <connections/>
</ui>
